apps/vr-tests-react-components/package.json

```diff
@@ -66,6 +66,7 @@
     "@fluentui/react-textarea": "*",
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31219&commitId=b79d2458999b1ff5fc2e11de6a8900e3a470b2f3&env=prod&clientType=FLUENTUI)


  <details><summary><b>TagPicker</b> 32 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TagPicker.appearance - Dark Mode.chromium.png|0| Added|
|TagPicker.appearance - High Contrast.chromium.png|0| Added|
|TagPicker.appearance - RTL.chromium.png|0| Added|
|TagPicker.appearance.chromium.png|0| Added|
|TagPicker.default - Dark Mode.chromium.png|0| Added|
|TagPicker.default - High Contrast.chromium.png|0| Added|
|TagPicker.default - RTL.chromium.png|0| Added|
|TagPicker.default open - Dark Mode.chromium.png|0| Added|
|TagPicker.default open - High Contrast.chromium.png|0| Added|
|TagPicker.default open - RTL.chromium.png|0| Added|
|TagPicker.default open.chromium.png|0| Added|
|TagPicker.default.chromium.png|0| Added|
|TagPicker.disabled - Dark Mode.chromium.png|0| Added|
|TagPicker.disabled - High Contrast.chromium.png|0| Added|
|TagPicker.disabled - RTL.chromium.png|0| Added|
|TagPicker.disabled.chromium.png|0| Added|
|TagPicker.expand icon - Dark Mode.chromium.png|0| Added|
|TagPicker.expand icon - High Contrast.chromium.png|0| Added|
|TagPicker.expand icon - RTL.chromium.png|0| Added|
|TagPicker.expand icon.chromium.png|0| Added|
|TagPicker.grouped - Dark Mode.chromium.png|0| Added|
|TagPicker.grouped - High Contrast.chromium.png|0| Added|
|TagPicker.grouped - RTL.chromium.png|0| Added|
|TagPicker.grouped.chromium.png|0| Added|
|TagPicker.secondary action - Dark Mode.chromium.png|0| Added|
|TagPicker.secondary action - High Contrast.chromium.png|0| Added|
|TagPicker.secondary action - RTL.chromium.png|0| Added|
|TagPicker.secondary action.chromium.png|0| Added|
|TagPicker.size - Dark Mode.chromium.png|0| Added|
|TagPicker.size - High Contrast.chromium.png|0| Added|
|TagPicker.size - RTL.chromium.png|0| Added|
|TagPicker.size.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>